Client/Ecosia/AppDelegate/Leaves/OrientationLockAppDelegate.swift

```diff
@@ -0,0 +1,18 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+final class OrientationLockAppDelegate: AppDelegateLeaf {
+    
+    private var orientationLock = UIInterfaceOrientationMask.all
+    
+    required init(orientationLock: UIInterfaceOrientationMask) {
+        self.orientationLock = orientationLock
+    }
+    
+    func application(_ application: UIApplication, supportedInterfaceOrientationsFor window: UIWindow?) -> UIInterfaceOrientationMask {
+        self.orientationLock
```

Comment:
This feels a bit strange to me given that `self.orientationLock` is even the default return on `EcosiaAppDelegate` 🤔 

I guess this is also a specific implementation not expected in any other leaf and wonder if it could follow a similar logic as the user activity and open url as mentioned in the other comment.

Reply:
Decided to leave the `self.orientationLock` there as its reference is used in `OrientationLockUtility`